frappe/core/page/background_jobs/background_jobs.py

```diff
@@ -44,15 +42,27 @@ def add_job(j, name):
 
 	for q in queues:
 		if q.name != 'failed':
-			for j in q.get_jobs(): add_job(j, q.name)
+			for j in q.get_jobs():
+				add_job(j, q.name)
 
 	if cint(show_failed):
 		for q in queues:
 			if q.name == 'failed':
```

Comment:
This shouldn't work anymore.

https://python-rq.org/docs/jobs/#failed-jobs

Reply:
Alright, it's using the RQ library instead to fetch from the failed job registry